experiments/data/sbatch_hf_split.sh

```diff
@@ -0,0 +1,26 @@
+#! /bin/bash
+#SBATCH --job-name="llched-split"
+#SBATCH --nodes=1
+#SBATCH --ntasks-per-node=1
+#SBATCH --cpus-per-task=12
+#SBATCH --output=/fsx/proj-chemnlp/experiments/logs/split_%j.out
+#SBATCH --error=/fsx/proj-chemnlp/experiments/logs/split_%j.err
+#SBATCH --open-mode=append
+#SBATCH --account=topchem
+#SBATCH --partition=g40x
+#SBATCH --exclusive
+
+### This script runs tokenisation of Hugging Face datasets
+### The first arg ($1) is the prefix directory where the environment is saved
```

Comment:
is $1 missing?

Reply:
oh yes, I had a short term hack without using the environment script. I've added comments and the same environment creation script now as tokenisation